cli/cmd/cmdutil/cmdutil.go

```diff
@@ -143,6 +143,21 @@ func InputPrompt(msg, def string) string {
 	return result
 }
 
+func MandatoryInputPrompt(msg string) string {
+	prompt := []*survey.Question{{
+		Name:      "name",
+		Prompt:    &survey.Input{Message: msg},
+		Validate:  survey.Required,
+		Transform: survey.Title,
```

Comment:
I don't think `survey.Title` is the behavior we expect? Most of these inputs take lowercase values, no?

Reply:
Lemme check what makes more sense